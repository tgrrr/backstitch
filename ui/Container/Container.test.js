import React from 'react'
import renderer from 'react-test-renderer'
import { Container } from '.'

it('Container: default', () => {
    const component = renderer.create(
        <Container>
            <>
                Container
            </>
        </Container>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
