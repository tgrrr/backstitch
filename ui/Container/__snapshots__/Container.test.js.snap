// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Container "
    data-testid="Container"
  >
    <div
      className="MuiContainer-root MuiContainer-maxWidthLg"
    >
      Container
    </div>
  </div>
</div>
`;
